avaframe/out3SimpPlot/outQuickPlot.py

```diff
@@ -109,13 +119,15 @@ def quickPlot(avaDir, suffix, cfg, simName):
         ax2.set_title(title)
         ax3 = fig.add_subplot(133)
         cmap = cmapdiv
-        im3 = plt.imshow(data1-data2, cmap=cmap,
+        im3 = plt.imshow(dataDiff, cmap=cmap,
                          extent=[0, Lx, 0, Ly], origin='lower', aspect=nx/ny)
         fig.colorbar(im3, ax=ax3)
+        ax3.text(nybox, nxbox, 'Mean: %.2f\n Max: %.2f\n Min: %.2f' % (diffMean, diffMax, diffMin), bbox=dict(boxstyle="square", ec='white', fc='white'),
+        horizontalalignment='left', verticalalignment='bottom')
```

Comment:
could we maybe add some units here? to be sure if its m or kPa or %

Reply:
same as above, then i think we have to introduce the units for the colorbars too

Reply:
Added the unit